drgn/internal/sudohelper.py

```diff
@@ -27,6 +27,8 @@ def open_via_sudo(
             subprocess.check_call(
                 [
                     "sudo",
+                    "-p",
+                    f"[sudo] password for %p to open {path}: ",
```

Comment:
Today I learned about `sudo -p` :)  Turns out it's present even in OL7/RHEL7 so it's been around for a while and should work on most distros.

Reply:
I hadn't even considered that, but it looks like it goes way back to sudo-project/sudo@337897e2d83489b4427a90e3dba20fad28d3fe88 in 1995! So I think we're covered ;)